documentation/book/assembly-upgrade-resources.adoc

```diff
@@ -0,0 +1,18 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+[id='assembly-upgrade-resources-{context}']
+= {ProductName} resource upgrades
+For this release of {ProductName}, resources that use the API version `{KafkaApiVersionPrev}` must be updated to use `{KafkaApiVersion}`.
```

Comment:
I don't think there's any value is using an attribute for the API version: These instructions are specific to the v1alpha1 to v1beta1 change.

Reply:
Will leave as is for now.